packages/components/src/breadcrumbs/index.tsx

```diff
@@ -7,13 +7,24 @@ import {
 import { useResizeObserver, useMergeRefs } from '@wordpress/compose';
 import { useI18n } from '@wordpress/react-i18n';
 import clsx from 'clsx';
-import React, { useState, forwardRef, useRef } from 'react';
+import React, { useState, forwardRef, useRef, useMemo } from 'react';
 import Menu from '../menu';
-import { BreadcrumbProps, BreadcrumbItemProps } from './types';
+import { BreadcrumbProps, BreadcrumbItemProps, Item } from './types';
 import './style.scss';
 
-function BreadcrumbsMenu( { items }: { items: BreadcrumbItemProps[] } ) {
+function defaultRenderItemLink( props: BreadcrumbItemProps ) {
+	return <a { ...props }>{ props.label }</a>;
+}
+
+function BreadcrumbsMenu( {
+	items,
+	renderItemLink,
```

Comment:
Should we include a unit test to cover the new prop?

Reply:
Is this [test](https://github.com/Automattic/wp-calypso/pull/103321/commits/41b389c07a7fb93461ada7fcb05df79a5de2969f) like what you had in mind?

Reply:
Yep, that does the job for me 👍 Especially considering that it demonstrates an example of providing a custom behavior and not custom design.